src/js/index.js

```diff
@@ -0,0 +1,130 @@
+import createHeader from './header';
+import createFooter from './footer';
+import createTextarea from './textarea';
+import * as board from './board';
+
+import '../css/main.scss';
+
+const state = {
+  lang: 'ru',
```

Comment:
You have two possible options for a language in your app: Russian and English. Usually in such cases enums are used. I'd recommend to use an object in this specific case, smth like:

`const LANG = {
  EN: 'en',
  RU: 'ru',
};`

and then

`const state = {
  lang: LANG.RU,
  count: 1,
  text: '',
  capsLock: false,
};`

Such approach will allow you to make sure the language is always either RU or EN and typos aren't possible - your IDE won't allow you to specify anything other than LANG.RU or LANG.EN

Reply:
done

Reply:
GitHub squeezes code comments into one line. Format your code properly:
![Clip2net_220604132426](https://user-images.githubusercontent.com/60316134/171995259-993b86a7-5ffd-4b98-95b3-b4dfc5ed433c.png)

Reply:
done